### PR TITLE
Asheron's Raiment changed due to data from pcap, log and wiki. 

### PR DESCRIPTION
**Pull request by LawrencePW**
Changed spells, arcane lore and wield requirement

### CODE REVIEW

**Comment by LawrencePW** [comment]
Removed the config files from pull. Rounded the float values.

File: .gitignore [hunk 1]
```diff
@@ -1,224 +0,0 @@
-#################
-## ACE-World Specific
-#################
-Database/compiled/password.txt
```
Comment (ziang4891) [inline .gitignore:4]: The .gitignore should be left.  It was the other files, presumably added by VS that weren't needed

File: Database/Patches/2013-09-TheQuestForFreedom/9 WeenieDefaults/Clothing/Clothing/24359 Asheron's Raiment.sql [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+     , (24359,   3,         61) /* PaletteTemplate - White */
+     , (24359,   4,        126) /* ClothingPriority - UnderwearUpperLegs, UnderwearLowerLegs, UnderwearChest, UnderwearAbdomen, UnderwearUpperArms, UnderwearLowerArms */
+     , (24359,   5,        800) /* EncumbranceVal */
+     , (24359,   8,         90) /* Mass */
+     , (24359,   9,        222) /* ValidLocations - ChestWear, AbdomenWear, UpperArmWear, LowerArmWear, UpperLegWear, LowerLegWear */
+     , (24359,  16,          1) /* ItemUseable - No */
+     , (24359,  19,       5000) /* Value */
+     , (24359,  27,          1) /* ArmorType - Cloth */
+     , (24359,  28,          0) /* ArmorLevel */
+     , (24359,  53,        101) /* PlacementPosition - Resting */
```
Comment (ziang4891) [inline Database/Patches/2013-09-TheQuestForFreedom/9 WeenieDefaults/Clothing/Clothing/24359 Asheron's Raiment.sql:17]: PlacementPosition - Resting is a default value that doesn't need to be in the file.  Same goes for all the other wcids in the PR

File: Database/Patches/2013-09-TheQuestForFreedom/9 WeenieDefaults/Clothing/Clothing/24359 Asheron's Raiment.sql [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+     , (24359, 106,        275) /* ItemSpellcraft */
+     , (24359, 107,        800) /* ItemCurMana */
+     , (24359, 108,        800) /* ItemMaxMana */
+     , (24359, 109,        100) /* ItemDifficulty */
+     , (24359, 158,          7) /* WieldRequirements - Level */
+     , (24359, 159,          1) /* WieldSkillType - Axe */
+     , (24359, 160,         60) /* WieldDifficulty */;
+
+INSERT INTO `weenie_properties_bool` (`object_Id`, `type`, `value`)
+VALUES (24359,  14, True ) /* GravityStatus */
```
Comment (ziang4891) [inline Database/Patches/2013-09-TheQuestForFreedom/9 WeenieDefaults/Clothing/Clothing/24359 Asheron's Raiment.sql:28]: Not needed in ACE, as it is provided by the PropertyInt.PhysicsState.  Same for the other wcids in this PR

File: .gitignore (hunk 1 above)
Comment (ziang4891) [reply .gitignore:4]: .gitignore needs to be the version of the file, with the same contents, as the one in Patches master.  VS replaced it with one of its own making that is missing everything that should be present in the file.

**Comment by DridsGhost** [comment]
I'd suggest just deleted/closing this PR and starting over with something cleaner.

**Pull request closed without merging**